free-courses-en.md

```diff
@@ -204,6 +204,7 @@
 * [Deep Learning Specialization](https://www.coursera.org/specializations/deep-learning) (coursera)
 * [Deep Reinforcement Learning](http://rail.eecs.berkeley.edu/deeprlcourse/) - Sergey Levine
 * [Intro to Deep Learning](https://www.kaggle.com/learn/intro-to-deep-learning) - Ryan Holbrook
+* [Introduction to Reinforcement Learning](https://www.youtube.com/watch?v=2pWv7GOvuf0) - David Silver
```

Comment:
We generally prefer to use playlists rather than videos, it'll also make it easier to find the next video and understand the length/scope of the course ahead of time.

```suggestion
* [Introduction to Reinforcement learning with David Silver](https://www.youtube.com/playlist?list=PLqYmG7hTraZBiG_XpjnPrSNw-1XQaM_gB) - David Silver
```